libzeth/snarks/groth16/groth16_snark.tcc

```diff
@@ -132,8 +132,10 @@ template<typename ppT>
 std::ostream &groth16_snark<ppT>::proof_write_json(
     const typename groth16_snark<ppT>::proof &proof, std::ostream &out_s)
 {
+    // JSON matches the protobuf format, where we export -b instead of b, to
+    // support efficient verification in contracts.
```

Comment:
Same as above. I'd be clear on the fact that these changes are mostly for simplicity matters (as per the main thread discussion)

Reply:
Added a note on the doc comments for this function.